File: alarmclock/src/main/java/com/externalalarmclock/alarmclock/healthchecks/AlarmClockHealthCheck.java
package com.externalalarmclock.alarmclock.healthchecks;

import com.codahale.metrics.health.HealthCheck;

public class AlarmClockHealthCheck extends HealthCheck {

	@Override
	protected Result check() throws Exception {
		return Result.healthy("Nothing to check yet.");
	}
}

File: alarmclock/src/main/java/com/externalalarmclock/alarmclock/healthchecks/AlarmClockHealthCheck.kt
package com.externalalarmclock.alarmclock.healthchecks

import com.codahale.metrics.health.HealthCheck

class AlarmClockHealthCheck : HealthCheck() {

    @Throws(Exception::class)
    public override fun check(): HealthCheck.Result {
        return HealthCheck.Result.healthy("Nothing to check yet.")
    }
}
